tests/system_tests/epics/adsim/test_adsim_system.py

```diff
@@ -35,17 +35,23 @@
 TIMEOUT = 10.0
 
 
-@pytest.fixture(scope="module", autouse=True)
+@pytest.fixture(scope="module")
 def with_env():
-    with patch.dict(
```

Comment:
Is there a reason this is preferable to patching the environment with clear? It seems more complex to me- was the old way not passing all required environment variables?